sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveInspectors.scala

```diff
@@ -32,6 +32,7 @@ import org.apache.spark.sql.AnalysisException
 import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.catalyst.util._
+import org.apache.spark.sql.execution.datasources.DaysWritable
```

Comment:
unnecessary change

Reply:
If I remove it, the build fails with:
```
Error:(1012, 44) not found: type DaysWritable
  private def getDateWritable(value: Any): DaysWritable =
```
I moved `DaysWritable` from `sql/hive` to `sql/core` to reuse it in ORC.